src/hypofuzz/frontend/src/components/TestTable.tsx

```diff
@@ -189,7 +189,12 @@ export function TestTable({ tests, onFilterChange }: Props) {
         data={sortedTests}
         row={row}
         mobileRow={mobileRow}
-        getKey={test => test.database_key}
+        // we'd like to use
+        //   getKey={test => test.database_key}
+        // here, but test is not guaranteed to have a databse_key set (if e.g.
+        // ADD_REPORTS arrives before ADD_TESTS), and I'm more worried about
+        // accidentally choosing an overlapping fallback key than trying to squeeze
+        // out performance here.
```

Comment:
the nursery fix might handle this?  Packet reordering is still possible on the wire, but I think logical reordering would require something weird with reconnections that we're already assuming doesn't happen.